internal/cluster/validations/validation_test.go

```diff
@@ -679,35 +681,35 @@ var _ = Describe("IPv6 support", func() {
 	}
 })
 
-var _ = Describe("Network amount and order", func() {
+var _ = Describe("Machine Network amount and order", func() {
 	tests := []struct {
-		element []*string
+		element []*models.MachineNetwork
 		valid   bool
 	}{
 		{
-			element: []*string{swag.String("1.2.5.0/24"), swag.String("1002:db8::/119")},
+			element: []*models.MachineNetwork{{Cidr: "1.2.5.0/24"}, {Cidr: "1002:db8::/119"}},
 			valid:   true,
 		},
 		{
-			element: []*string{swag.String("1002:db8::/119"), swag.String("1.2.5.0/24")},
+			element: []*models.MachineNetwork{{Cidr: "1002:db8::/119"}, {Cidr: "1.2.5.0/24"}},
 			valid:   false,
 		},
 		{
-			element: []*string{swag.String("1.2.5.0/24"), swag.String("1002:db8::/119"), swag.String("1.2.6.0/24"), swag.String("1.2.7.0/24")},
-			valid:   true,
+			element: []*models.MachineNetwork{{Cidr: "1.2.5.0/24"}, {Cidr: "1002:db8::/119"}, {Cidr: "1.2.6.0/24"}, {Cidr: "1.2.7.0/24"}},
```

Comment:
For me and others coming here later, it would be helpful to comment in some way why these are valid or invalid.

Is this one invalid because only two are allowed?

Reply:
Fixed